apidoc/Titanium/UI/ProgressBar.yml

```diff
@@ -71,13 +71,13 @@ properties:
   - name: message
     summary: Progress bar text.
     type: String
-
+  - name: tintColor
+    summary: The color shown for the portion of the progress bar that is filled.
```

Comment:
When you redefine the property, it will (as far as I know) override the "parent" key.